components/Composition/FAnnouncementCard.jsx

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet, View } from 'react-native';
+import { FImage } from 'components/Composition/FImage';
+import { getHalfBorderRadius } from 'utils/getHalfBorderRadius';
+import colors from 'themes/colors';
+import sizes from 'themes/sizes';
+import opacities from 'themes/opacities';
+import fonts from 'themes/fonts';
+import { FHeading } from 'components/Composition/FHeading';
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native-web';
+import { FHeadingWithIcon } from 'components/Composition/FHeadingWithIcon';
+import placements from 'themes/placements';
+import icons from 'themes/icons';
+
+export const FAnnouncementCard = ({ width, data }) => {
+  const cropText = (text, number) => `${text.slice(0, width / number)}...`;
+  return (
+    <TouchableWithoutFeedback>
+      <View style={{
+        width,
+        ...styles.container,
+      }}
+      >
+        <FImage
+          networkImageUrl={data.photoURL}
+          imageStyle={styles.image}
+          containerStyle={styles.imageContainer}
+        />
+        <FHeading
+          title={cropText(data.title, 12)}
+          size={fonts.HEADING_MEDIUM}
+          weight={fonts.HEADING_WEIGHT_BOLD}
+          marginBottom={sizes.MARGIN_5}
+        />
+        <FHeading
+          title={cropText(data.description, 5)}
+          size={fonts.HEADING_NORMAL}
+          color={colors.DARK_GRAY}
+          marginBottom={sizes.MARGIN_5}
+        />
+        <FHeadingWithIcon
+          titleStyle={{ marginLeft: sizes.MARGIN_5 }}
+          icon={icons.LOCATION_OUTLINE}
+          iconColor={colors.PRIMARY}
+          title={data.locationName}
+          iconPlacement={placements.LEFT}
+          titleColor={colors.DARK_GRAY}
+          containerStyle={placements.LEFT}
+        />
+        <FHeadingWithIcon
+          titleStyle={{ marginLeft: sizes.MARGIN_5 }}
+          icon={icons.CALENDAR}
+          iconColor={colors.PRIMARY}
+          title={data.date}
+          iconPlacement={placements.LEFT}
+          titleColor={colors.DARK_GRAY}
+          containerStyle={placements.LEFT}
+        />
+      </View>
+    </TouchableWithoutFeedback>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    height: 'auto',
```

Comment:
nie moga mieć height auto. Każdy card ma mieć taką samą wysokość bez względu na to co w sobie ma. Dac maxHeight i minHeight na takie same wartości, czyli wzorując sie na figmie. Przez to mogą wywalić CI się inne style np odległosci elementów w cardzie wiec to musisz posprawdzac jak sie zachowują

Reply:
no sprawdzałam i dlatego wycięłam tekst, tytuł ma mieć max 1 linike i tak samo z resztą, jak ekran będzie się zmieniał daje to spoko rwd tylko trzeba zapanować nad tym co jest w karcie w środku